source/patterns/@aws-solutions-constructs/core/lib/dynamodb-table-helper.ts

```diff
@@ -71,6 +71,30 @@ export function buildDynamoDBTable(scope: Construct, props: BuildDynamoDBTablePr
   }
 }
 
+export function checkTableProps(props: BuildDynamoDBTableProps) {
```

Comment:
I missed part of this earlier - this ensures there isn't more than one, but it doesn't ensure there is at least one.